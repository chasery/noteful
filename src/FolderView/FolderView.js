import React, { Component } from "react";

class FolderView extends Component {
  render() {
    return <div>FolderView</div>;
  }
}

export default FolderView;
