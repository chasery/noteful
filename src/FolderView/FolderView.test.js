import React from "react";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";
import FolderView from "./FolderView";

describe("FolderView component", () => {
  it("renders without error", () => {
    const wrapper = shallow(<FolderView />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
