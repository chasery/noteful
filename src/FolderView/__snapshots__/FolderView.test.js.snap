// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FolderView component renders without error 1`] = `
<div>
  FolderView
</div>
`;
